src/main/java/net/minecraftforge/server/permission/PermissionAPI.java

```diff
@@ -19,80 +19,73 @@
 
 package net.minecraftforge.server.permission;
 
-import com.google.common.base.Preconditions;
-import com.mojang.authlib.GameProfile;
-import net.minecraft.world.entity.player.Player;
-import net.minecraftforge.server.permission.context.IContext;
-import net.minecraftforge.server.permission.context.PlayerContext;
-
-import javax.annotation.Nullable;
-
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.util.StringRepresentable;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.server.permission.events.PermissionGatherEvent;
+import net.minecraftforge.server.permission.handler.DefaultPermissionHandler;
+import net.minecraftforge.server.permission.handler.IPermissionHandler;
+import net.minecraftforge.server.permission.nodes.PermissionDynamicContext;
+import net.minecraftforge.server.permission.nodes.PermissionNode;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-public class PermissionAPI
-{
-    private static final Logger LOGGER = LogManager.getLogger();
-    
-    private static IPermissionHandler permissionHandler = DefaultPermissionHandler.INSTANCE;
+import java.util.Collection;
 
-    /**
-     * <b>Only use this in PreInit state!</b>
-     */
-    public static void setPermissionHandler(IPermissionHandler handler)
-    {
-        Preconditions.checkNotNull(handler, "Permission handler can't be null!");
-        // TODO Loader states Preconditions.checkState(Loader.instance().getLoaderState().ordinal() <= LoaderState.PREINITIALIZATION.ordinal(), "Can't register after IPermissionHandler PreInit!");
-        LOGGER.warn("Replacing {} with {}", permissionHandler.getClass().getName(), handler.getClass().getName());
-        permissionHandler = handler;
-    }
+public final class PermissionAPI
+{
+   private static Logger LOGGER = LogManager.getLogger();
+   private static IPermissionHandler activeHandler = new DefaultPermissionHandler();
 
-    public static IPermissionHandler getPermissionHandler()
-    {
-        return permissionHandler;
-    }
+   public static Collection<PermissionNode<?>> getRegisteredNodes()
+   {
```

Comment:
Bracket on the wrong line.

Reply:
Formatting was correct after all. Resolved on the Discord Server.
<https://canary.discord.com/channels/313125603924639766/852298000042164244/883245371013337148>